tls/s2n_prf.c

```diff
@@ -453,22 +453,34 @@ int s2n_prf_calculate_master_secret(struct s2n_connection *conn, struct s2n_blob
     POSIX_ENSURE_EQ(s2n_conn_get_current_message_type(conn), CLIENT_KEY);
 
     /* TODO: https://github.com/aws/s2n-tls/issues/2990 */
-    if (conn->ems_negotiated && s2n_in_unit_test()) {
+    if (conn->ems_negotiated && S2N_IN_TEST) {
```

Comment:
If you want this method to look a little cleaner, consider bailing early instead of nested if/else.